### PR TITLE
Some fixed

### PR DESCRIPTION
**Pull request by buster95**

### CODE REVIEW

File: config/example.env [hunk 1]
```diff
@@ -1,4 +0,0 @@
-REDIS_HOST=redis
-REDIS_PORT=6379
-REDIS_PASSWORD=123456
-SERVER_PORT=3000
```
Comment (ebwinters) [inline config/example.env:4]: Why remove the example config? We may need it for dev setup instructions in the future.

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

Comment (buster95) [reply config/example.env:4]: Because we have two example files one in the project root and this

**Pull request merged**